plugin.video.ustvnow/resources/lib/net.py

```diff
@@ -252,16 +265,21 @@ def _fetch(self, url, form_data={}, headers={}, compression=True):
             meta-information about the page and the page content.
         '''
         encoding = ''
-        req = urllib2.Request(url)
-        if form_data:
-            form_data = urllib.urlencode(form_data)
-            req = urllib2.Request(url, form_data)
-        req.add_header('User-Agent', self._user_agent)
-        for k, v in headers.items():
-            req.add_header(k, v)
-        if compression:
-            req.add_header('Accept-Encoding', 'gzip')
-        response = urllib2.urlopen(req)
+        try:
+            req = urllib2.Request(url)
+            if form_data:
+                form_data = urllib.urlencode(form_data)
+                req = urllib2.Request(url, form_data)
+            req.add_header('User-Agent', self._user_agent)
+            for k, v in headers.items():
+                req.add_header(k, v)
+            if compression:
+                req.add_header('Accept-Encoding', 'gzip')
+            response = urllib2.urlopen(req)
+        except urllib2.HTTPError, e:
```

Comment:
`except urllib2.HTTPError as e` for python3 compatibility (https://www.python.org/dev/peps/pep-3110/#compatibility)

Reply:
Thanks that document says either works? except_clause: 'except' [test [('as' | ',') test]]

Reply:
`except_clause: 'except' [test [('as' | ',') test]]` is what python 2.6 and python 2.7 accept
python3 only accepts `except_clause: 'except' [test ['as' NAME]]`

Reply:
ahhh, Duh :) thanks for the link, it will be helpful in the future.